ADOPTERS.md

```diff
@@ -64,6 +64,11 @@
+<<<<<<< HEAD
 | [Signal Iduna Group](https://www.signal-iduna.de/)                       | [Jonas Thomsen](https://github.com/JoThomsen)                                                                                                                                                                | Developer Portal, documentation, monitoring, service catalog for our insurance ecosystem                                                                                                                                                                                                                                                             |
 | [Tradeshift](https://www.tradeshift.com/)                                | [Soren Mathiasen](https://github.com/sorenmat)                                                                                                                                                               | Developer Portal: documentation, monitoring, service templates, service catalog for our micro services                                                                                                                                                                                                                                               |
 | [Unity](https://unity.com) | [Ted Cordery](https://github.com/TeddyBallGame) | A centralized service catalog with documentation for our service engineers. |
+=======
+| [Signal Iduna Group](https://www.signal-iduna.de/)                                                 | [Jonas Thomsen](https://github.com/JoThomsen)                                                                                                                                                                           | Developer Portal, documentation, monitoring, service catalog for our insurance ecosystem                                                                                                                                                                      
+| [PicPay](https://www.picpay.com)                   | [Luis Baroni](https://github.com/lcsbaroni), [Renata Poluceno](https://github.com/renatapoluceno), [PicPay](https://github.com/picpay)                                                | Developer portal for building services throught templates, service catalog with ownership of services, documentation and metrics providing autonomy and visibility for all.                                                                                                                                                                                                                        |
+>>>>>>> Add PicPay as adopter
```

Comment:
Looks like a bit of a merge conflict here that needs fixing 😅

Reply:
Sorry. I forgot a CTRL+S 🤦‍♂️.